@here/harp-mapview/lib/MapView.ts

```diff
@@ -1717,12 +1718,19 @@ export class MapView extends THREE.EventDispatcher {
         this.m_screenProjector.update(this.m_camera, this.m_worldCenter, width, height);
         this.m_screenCollisions.update(width, height);
 
-        this.m_zoomLevel = MapViewUtils.calculateZoomLevelFromHeight(
-            Math.abs(this.m_camera.position.z),
-            this
+        this.m_pixelToWorld = undefined;
+
+        const downQuat = this.m_tempQuat.setFromAxisAngle(EYE_INVERSE, -this.camera.rotation.z);
```

Comment:
Why storing the quaternion in the downQuat local variable? Use ``this.m_tempQuat`` afterwards instead.

Reply:
@amsgames94 
I would argue that this '`downQuat` is more readable and proper name for this particular variable in this context.
Using it as `tmpQuat` doesn't help understanding code at all.